curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/copy-array-items-using-slice.english.md

```diff
@@ -8,7 +8,7 @@ forumTopicId: 301158
 
 ## Description
 <section id='description'>
-The next method we will cover is <code>slice()</code>. <code>slice()</code>, rather than modifying an array, copies, or <em>extracts</em>, a given number of elements to a new array, leaving the array it is called upon untouched. <code>slice()</code> takes only 2 parameters &mdash; the first is the index at which to begin extraction, and the second is the index at which to stop extraction (extraction will occur up to, but not including the element at this index). Consider this:
+The next method we will cover is <code>slice()</code>. Rather than modifying an array,<code>slice()</code> copies or <em>extracts</em> a given number of elements to a new array, leaving the array it is called upon untouched. <code>slice()</code> takes only 2 parameters &mdash; the first is the index at which to begin extraction, and the second is the index at which to stop extraction (extraction will occur up to, but not including the element at this index). Consider this:
```

Comment:
@JovaniTarnowski, thank you for submitting a fix for this. Just noticed that we need a space here:

`Rather than modifying an array, <code>slice()</code> copies or <em>extracts</em> a given number of elements to a new array, leaving the array it is called upon untouched.`

Reply:
@scissorsneedfoodtoo Thank you for your feedback, I'm going to fix it.